testnet/stacks-node/src/config.rs

```diff
@@ -408,6 +408,33 @@ impl Config {
             burnchain.pox_constants.v1_unlock_height = v1_unlock_height;
         }
 
+        if let Some(epochs) = &self.burnchain.epochs {
+            // Iterate through the epochs vector and find the item where epoch_id == StacksEpochId::Epoch23
+            if let Some(epoch) = epochs
+                .iter()
+                .find(|epoch| epoch.epoch_id == StacksEpochId::Epoch23)
+            {
+                // Override v2_unlock_height to the start_height of epoch2.3
+                debug!(
+                    "Override v2_unlock_height from {} to {}",
+                    burnchain.pox_constants.v2_unlock_height, epoch.start_height
+                );
+                burnchain.pox_constants.v2_unlock_height = epoch.start_height as u32;
+            }
+
+            if let Some(epoch) = epochs
+                .iter()
+                .find(|epoch| epoch.epoch_id == StacksEpochId::Epoch24)
+            {
+                // Override pox_3_activation_height to the start_height of epoch2.4
+                debug!(
+                    "Override pox_3_activation_height from {} to {}",
+                    burnchain.pox_constants.pox_3_activation_height, epoch.start_height
+                );
+                burnchain.pox_constants.pox_3_activation_height = epoch.start_height as u32;
```

Comment:
Same here, unclear to me if off-by-one

Reply:
Here there's no need for a +1 because the pox_3_activation_height is set to the epoch start height.